tests/measurements/test_vn_entropy.py

```diff
@@ -0,0 +1,37 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Unit tests for the vn_entropy module"""
+import numpy as np
+
+import pennylane as qml
+
+
+class TestVnEntropy:
```

Comment:
Probably worth having a more complicated test with non-zero Von Neumann entropy. Also worth adding a test specifically to check the type of the measurement process and its methods respectively.

Reply:
Done!